source/fuzz/protobufs/spvtoolsfuzz.proto

```diff
@@ -1480,6 +1481,20 @@ message TransformationReplaceLoadStoreWithCopyMemory {
   InstructionDescriptor store_instruction_descriptor = 2;
 }
 
+message TransformationReplaceOpSelectWithConditionalBranch {
+
+  // A transformation that takes an OpSelect instruction with a
+  // scalar boolean condition and replaces it with a conditional
+  // branch and an OpPhi instruction.
```

Comment:
I wonder whether it might be nice to have two block fields: true_block and false_block, and to allow at most one of them to be 0.  If neither are 0 we end up with a proper diamond shape, and otherwise we get a half diamond, and it can be in either direction -- see what I mean?  The fuzzer pass could decide which to create.  This might add a bit more diversity.

Reply:
Done